### PR TITLE
Enhancement/fe touchups

### PR DESCRIPTION
**Pull request by DemogorGod**
@shanejearley here you go, updated the landing page and cleaned up some stuff. obviously the footer and any other links do not existent right now, however, they will just not take the user anywhere for the moment until we build out those pages.

### CODE REVIEW

File: apps/landing/src/pages/landing/components/Content.vue [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        class="text-center"
+      >
+        <span class="text-[12px] font-[600] text-primary">Assurance </span>
+        <h1 class="text-[28px] font-[600] leading-10 mb-[10px] tracking-tight text-grey_3">
+          Minimal Trust
+        </h1>
+
+        <div class="flex items-center justify-center">
+          <p class="text-[14px] font-[400] text-grey_4 w-full max-w-[650px] tracking-wide">
+            With Casimir's native integration at Layer 1, users
```
Comment (shanejearley) [inline apps/landing/src/pages/landing/components/Content.vue:100]: @natemiller1 _native integration at Layer 1_ is unspecific and can basically mean smart contract. This paragraph also doesn't seem to focus on "Minimal Trust" – the content says about simple access with high utility. We should probably have a section for both. Do you want to try spinning this into two paragraphs with a little more specificity?

**Review by shanejearley — CHANGES_REQUESTED** [review]

File: apps/landing/src/pages/landing/components/Content.vue (hunk 1 above)
Comment (natemiller1) [reply apps/landing/src/pages/landing/components/Content.vue:100]: What about something like this:

Casimir's smart contract primitives and decentralized oracle network create a minimal trust environment to reduce counter-party risk and centralized points of failure. This allows users to create safe, non-custodial interactions not only with staking but other DeFi tools across Web3.

As we grow, users will be able to use Casimir to access a variety of use cases beyond staking. Shortly after launch, we will enable restaking on Eigenlayer through the creation of non-custodial Eigenpods. This allows users to utilize Eigenlayer's rehypothecation without worrying about the cap limits of LSTs

Comment (shanejearley) [reply apps/landing/src/pages/landing/components/Content.vue:100]: Looks great! Succinct and begs the right questions for future sections to answer more in depth.

Small changes:

- "...permissionless smart contracts, distributed key generation, and decentralized oracles..."

- Following their naming, we should use "EigenLayer" and "EigenPod"

File: apps/landing/src/pages/landing/components/Content.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            can easily engage with NFTs or DeFi protocols, whether it's creating, trading, or managing
+            these digital assets. By eliminating the need for additional bridges or intermediaries,
+            Casimir enables a more direct and efficient interaction with the NFT and DeFi space. For
+            emerging use cases, such as Eigenlayer, Casimir's integration at Layer 1 would enable users
```
[the remark this answers is not in the document]
Comment (shanejearley) [reply apps/landing/src/pages/landing/components/Content.vue:104]: Resolving in respect to Nate's full paragraph edit below.

File: apps/landing/src/pages/landing/components/Content.vue (hunk 1 above)
Comment (natemiller1) [reply apps/landing/src/pages/landing/components/Content.vue:100]: Ok let's go with: 

Casimir's smart contract primitives and decentralized oracle network create a trust-minimized environment to reduce counter-party risk and other centralized points of failure. This allows users to create safe, non-custodial interactions not only with staking but other DeFi tools across Web3.

As we grow, users will be able to use Casimir to access a variety of use cases beyond staking. Shortly after launch, we will enable restaking on EigenLayer through the creation of non-custodial EigenPods. This allows users to utilize EigenLayer's rehypothecation without worrying about the cap limits of LSTs

**Comment by ccali11** [comment]
@shanejearley @DemogorGod - taking a look at this now; will add a focus on UI copy review 👍

**Pull request closed without merging**